src/compas/__init__.py

```diff
@@ -275,9 +275,10 @@ def safe_extract(tar, path=".", members=None, numeric_owner=False):
     return bunny
 
 
-def devday(year, is_open):
+def devday(year, *, open):
```

Comment:
yep, let's change it to 
```suggestion
def devday(year, open=True):
```

Reply:
Sure, let's do it this way!